monitored-httpclient/src/test/java/com/github/nhenneaux/resilienthttpclient/monitoredclientpool/ResilientClientTest.java

```diff
@@ -389,55 +393,45 @@ void shouldAddFailureCountByWhenStatusCodeIsNotSuccess() throws IOException, URI
         final ServerConfiguration serverConfiguration = new ServerConfiguration(hostname);
 
         try (final HttpClientPool httpClientPool = HttpClientPool.builder(serverConfiguration).build()) {
+            final int serverErrorCode = ThreadLocalRandom.current().nextInt(500, 600);
```

Comment:
I would avoid random error code since it could make the test more unstable if 550-600 for ex is failing differently

Reply:
it needs to return the same code back so it is not success. Still failing with it